scripts/perftests/testCases/Test-Roslyn.ps1

```diff
@@ -0,0 +1,28 @@
+Param(
+    [Parameter(Mandatory=$true)]
+    [string]$nugetClientFilePath,
+    [Parameter(Mandatory=$true)]
+    [string]$sourceRootDirectory,
+    [Parameter(Mandatory=$true)]
+    [string]$resultsDirectoryPath,
+    [Parameter(Mandatory=$true)]
+    [string]$logsPath
+)
+
```

Comment:
If this test case is not needed, consider removing the file instead of leaving it commented out.